AirSkyBoat/AirSkyBoat
# Creates weaponskill module for era weaponskills

**_I affirm:_**


## Please enter a player-facing description
Nothing to say


## What does this pull request do? (Please be technical)
Modularization of all weaponskill luas


## Steps to test these changes
Use every single weaponskill in the game and see no errors

I have tested every weaponskills no errors - verified that it does read the module


## Special Deployment Considerations
Make sure you have the init.txt run this file.
